apps/fabric-website/src/components/App/AppState.tsx

```diff
@@ -792,7 +792,7 @@ export const AppState: IAppState = {
         },
         {
           title: 'Customization',
-          url: '#/components/customization',
+          url: '#/components/customizations',
```

Comment:
Just curious, why don't we change this to '#/customizations' instead?

Reply:
So I'd have to make some structural changes to this that I'm not entirely comfortable with on shield i suppose.

Reply:
Oh I see, because of the `className: 'componentsPage'`?